clients/admin-ui/src/features/config-wizard/ScannerError.tsx

```diff
@@ -36,7 +42,7 @@ const ScannerError = ({ error }: { error: ParsedError }) => (
       </Heading>
     </HStack>
 
-    {error.status === 403 ? (
+    {error.status === 403 && isAWSScan ? (
```

Comment:
This kind of boolean check is generally a code smell, as it provides the minimum amount of information and tends to spawn more and more special cases. Passing in the "scan type/target" is a bit better, using the `ValidTargets` enum which is usually whats going to determine which text should change.

Reply:
Yeah, I went back and forth on this, but passing in the scan type does seem like a better approach.